sdcm/tester.py

```diff
@@ -2625,6 +2625,10 @@ def _get_common_email_data(self) -> dict:
             self.log.error(f"Can't discover instance type for {backend}!")
             scylla_instance_type = "N/A"
         job_name = os.environ.get('JOB_NAME').split("/")[-1] if os.environ.get('JOB_NAME') else config_file_name
+        nodes_shards = []
+        if self.params.get('db_nodes_shards_selection') == 'random':
```

Comment:
let's have it always not only when shards are random

Reply:
done